source/dyaml/dumper.d

```diff
@@ -59,7 +60,7 @@ import dyaml.tagdirective;
  * Write to memory:
  * --------------------
  * import std.stream;
- * auto stream = new MemoryStream();
```

Comment:
yes the solution is not pretty, but from what I saw introducing ranges into the library is a big task.

as you said, for now I think it is ok